### PR TITLE
Field trip august 2024 fixes

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
Related to the plan functionality. The plan button was causing confusion and problems so we got rid of it when field trip is open. Then we had the problem that the location changes were getting wiped away. We solved this by adding a confirm before the locations are overriden. The issue with this approach is that we will not save the updated location to the trip. Maybe that's ok?

### CODE REVIEW

File: lib/components/narrative/default/default-itinerary.js [hunk 1]
```diff
@@ -166,7 +166,7 @@ const ITINERARY_ATTRIBUTES = [
             includeSeconds={false}
           />
           <LegIconWrapper noSpace>
-            <LegIcon leg={leg} size={5} />
+            <LegIcon leg={{ ...leg, mode: 'WALK' }} size={5} />
```
Comment (daniel-heppner-ibigroup) [inline lib/components/narrative/default/default-itinerary.js:169]: does legIcon even need the whole leg?

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Looks fine to me

File: lib/components/narrative/default/default-itinerary.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/components/narrative/default/default-itinerary.js:169]: yeah it complains if the start/end stuff is missing

File: lib/components/app/call-taker-panel.js [hunk 2]
```diff
@@ -153,10 +153,12 @@ class CallTakerPanel extends Component {
     }
   }
 
+  // eslint-disable-next-line complexity
```
Comment (josh-willis-arcadis) [inline lib/components/app/call-taker-panel.js:156]: I've seen this warning in other places. Should be disable those as well?

**Review by josh-willis-arcadis — APPROVED** [review]
LGTM

File: lib/components/app/call-taker-panel.js (hunk 2 above)
Comment (miles-grant-ibigroup) [reply lib/components/app/call-taker-panel.js:156]: No! It's a good reminder to make our code better

**Pull request merged**